src/Microsoft.Azure.WebJobs.Host/Executors/DefaultStorageCredentialsValidator.cs

```diff
@@ -41,24 +41,32 @@ internal class DefaultStorageCredentialsValidator : IStorageCredentialsValidator
         {
             // Verify the credentials are correct.
             // Have to actually ping a storage operation.
-            IStorageBlobClient client = account.CreateBlobClient();
+            IStorageQueueClient client = account.CreateQueueClient();
 
             try
             {
                 // This can hang for a long time if the account name is wrong. 
                 // If will fail fast if the password is incorrect.
-                await client.GetServicePropertiesAsync(cancellationToken);
+                await client.SdkObject.GetServicePropertiesAsync(cancellationToken);
```

Comment:
Is this because our StorageQueueClient impl. doesn't implement GetServicePropertiesAsync? How hard would that be to add, so you don't have to access SdkObject here?